test/suites/dev/registar/test_utils_rpc.ts

```diff
@@ -0,0 +1,128 @@
+import { describeSuite, expect, beforeAll} from "@moonwall/cli";
+import { setupLogger } from "@moonwall/util";
+import { ApiPromise, Keyring, WsProvider } from "@polkadot/api";
+import { jumpSessions } from "../../../util/block";
+const fs = require('fs/promises');
+
+import "@polkadot/api-augment";
+
+describeSuite({
+  id: "D07",
+  title: "Test utils RPC",
+  foundationMethods: "dev",
+  testCases: ({ it, context, log }) => {
+    let polkadotJs: ApiPromise;
+    const anotherLogger = setupLogger("anotherLogger");
+    let alice, bob;
+    beforeAll(() => {
+      const keyring = new Keyring({ type: 'sr25519' });
+      alice = keyring.addFromUri('//Alice', { name: 'Alice default' });
+      bob = keyring.addFromUri('//Bob', { name: 'Bob default' });
+      polkadotJs = context.polkadotJs();
+    });
+
+    it({
+        id: "E01",
+        title: "Read a ChainSpec, convert it to ContainerChainGenesisData, and back to the same ChainSpec",
+        test: async function () {
+            // Create a new provider with extended rpc and types
+            const addr = polkadotJs._options.provider.endpoint;
+            const wsProvider = new WsProvider(addr);
+            const polkadotJs2 = await ApiPromise.create({ provider: wsProvider, types: {
+              ContainerChainGenesisData: {
+                storage: "Vec<ContainerChainGenesisDataItem>",
+                name: "Bytes",
+                id: "Bytes",
+                fork_id: "Option<Vec<u8>>",
+                extensions: "Bytes",
+                properties: "Properties",
+              },
+              Properties: {
+                token_metadata: "TokenMetadata",
+                is_ethereum: "bool",
+              },
+              TokenMetadata: {
+                // TODO: this is actually a Vec<u8>, but that doesn't work because polkadot.js converts the
+                // Vec<u8> into hex bytes, while the Rust code doesn't work with hex bytes because this is
+                // actually a BoundedVec, and there is no easy way to serialize a BoundedVec as hex bytes.
+                // Ideally this should simply be a string, because this is a token name like "UNIT".
+                token_symbol: "Vec<u16>",
```

Comment:
Cant this be bytes?

Reply:
Also I tried `Bytes` and `Vec<u8>` to see if one of them works, but they are exactly the same, both give the same error.

Reply:
For now let's leave it like this then, lets try to remember this for the future

Reply:
we can also see check how polkadotjs handles boundedVecs